src/views/HowToPlay/Home/Home.tsx

```diff
@@ -0,0 +1,20 @@
+import { KeepReading, BackButton } from '@ui/components';
+import { Outlet } from 'react-router';
+
+export const HowToPlay = () => {
```

Comment:
The name of this view component doesn't match the filename of `Home`. The code will work fine, but this could be confusing for us later on.